modules/output/xml/main.nf

```diff
@@ -219,7 +217,7 @@ def addMatchNode(String proteinMd5, Map match, def xml) {
             )
         }
 
-        if (memberDB == "panther") {
+        if (match.source == "InterPro" && appl == "panther") {
             xml."model-ac"(match.treegrafter.subfamilyAccession ?: match.modelAccession)
             match.treegrafter.goXRefs.each { goXref ->
                 xml."go-xref"(
```

Comment:
Why only report the Panther GoXrefs if InterPro-N is used, should we not report these when Panther is used without InterPro-N?